doc/HWReleaseProcess.md

```diff
@@ -19,10 +19,11 @@
 
 ## Major Releases
 
-1. The RTL team will consume the **release collateral** generated by the **Nightly Workflow** and verify it in the DV environment. The verification should include RTL synthesis using the chosen tool from the Caliptra WG.
-2. Upon success of the DV verification a new release will be made on the **caliptra** repo
-    -  A ZIP file containing the release collateral will be included as part of the release.
-4. The **caliptra-rtl** and **caliptra-sw** repos will be tagged with a **release** version tag.
+1. Before finalizing an RTL release, the hardware team will follow the procedure described in [CaliptraReleaseChecklist.md](https://github.com/chipsalliance/caliptra-rtl/blob/cc37991e50f9213219d13f2e0d99b0ba5c7460e0/docs/CaliptraReleaseChecklist.md).
```

Comment:
TODO: Before this PR is merged, we need to complete https://github.com/chipsalliance/caliptra-rtl/pull/496 and update the hyperlink to CaliptraReleaseChecklist.md

Reply:
Done